tests/models/litter/test_litter_model.py

```diff
@@ -272,4 +338,28 @@ def test_update(litter_model_fixture, dummy_litter_data):
     assert np.allclose(
         dummy_litter_data["litter_pool_above_structural"], end_above_struct
     )
+    assert np.allclose(dummy_litter_data["litter_pool_woody"], end_woody)
+    assert np.allclose(dummy_litter_data["litter_pool_below_metabolic"], end_below_meta)
+    assert np.allclose(
+        dummy_litter_data["litter_pool_below_structural"], end_below_struct
+    )
     assert np.allclose(dummy_litter_data["litter_C_mineralisation_rate"], c_mineral)
+
+
+def test_convert_soil_moisture_to_water_potential(
+    dummy_litter_data, top_soil_layer_index
+):
+    """Test that function to convert soil moisture to a water potential works."""
+    from virtual_rainforest.models.litter.litter_model import (
+        convert_soil_moisture_to_water_potential,
+    )
+
+    expected_potentials = [-568.76270, -50.105649, -152.49040]
```

Comment:
These values seem a bit low, did you check that you used the right units for the result to be in kPa? I thought soil water potential was more in the range of 0 to -200 or so for soil moisture above wilting point, but I might be wrong

Reply:
So, the soil moisturise used to generate this are [0.5, 0.7, 0.6]. I'm guessing that 0.5 would still be expected to be above wilting point? I think something is definitely wrong in the expression, as you would expect the highest soil moisture to result in the most negative soil water potential right?

Reply:
I think high soil moisture has the least negative potential because it is easier to remove water, right? So the direction seems right, maybe it's just the parameter you used in the equation

Reply:
The decrease is steep at some point, so you might be right

Reply:
I'll look into it a bit more because it is very steep. Like a soil moisture of 0.1 results in a potential of -411224.11 kPa, which feels very high?

Reply:
If you look at the curves, they are very flat when soil moisture is high and then increase exponentially at low soil moisture, so that might be correct but worth double-checking